docs/api_specification.md

```diff
@@ -974,6 +974,22 @@ Content-Type: application/json
 
 -----------------------------------------------------------------------------------------------------------
 
+### POST /v1/api/services/`{serviceExternalId}`/send-live-email
+
+This endpoint will send an email to the user who signed the agreement with GOV.UK Pay for the service informing them that their service is now live.
```

Comment:
Maybe reference the endpoint that’s used to record the email address of the user who signed the agreement here?